react-ui/package.json

```diff
@@ -33,26 +28,39 @@
     "d3": "^7.3.0",
     "egeria-js-commons": "github:odpi/egeria-js-commons",
     "elkjs": "^0.7.1",
-    "happi-graph": "github:sarbull/happi-graph-1#happi-graph-in-react",
+    "happi-graph": "github:sarbull/happi-graph#happi-graph-in-react",
```

Comment:
shoudn't this point to the odpi repo?

Reply:
yes, it should, thanks

Reply:
@marius-patrascu i need to merge the [odpi/happi-graph#](https://github.com/odpi/happi-graph/pull/34) PR in happi-graph first, then come back here  and change it to `github:odpi/happi-graph#react`